mkt/urls.py

```diff
@@ -38,6 +39,8 @@
         name='detail.manifest'),
     url('^langpack/%s/manifest.webapp$' % mkt.ADDON_UUID,
         mini_langpack_manifest, name='langpack.manifest'),
+    url('^extension/(?P<uuid>[0-9a-f]{32})/manifest.json$',
```

Comment:
no particular reason, didn't feel like it was worth including patterns here just for one url. And we do need to include it from the root, the only other inclusion we have right now is prefixed with `/api/v2/`